smart-contracts/contracts/interfaces/IUnlock.sol

```diff
@@ -21,20 +21,23 @@ interface IUnlock
   function proxyAdminAddress() external view;
```

Comment:
Are changes in this file expected?

Reply:
We cannot change create lock.. the Unlock contract has already been deployed and this would break other people's integrations :(

Reply:
hmmm yes it was just to be consistent with what is already in the repo. Presently the Unlock contract in the repo has already been updated to use upgradeable locks, so I thought we may as well update the interface too - and use  IUnlockV8 where needed

Reply:
Yes, but that is actually the thing I wanted to chat about with you regarding the upgrade :/ Let's circle back via voice.